packages/dd-trace/src/config.js

```diff
@@ -99,6 +99,11 @@ class Config {
       process.env.DD_TRACE_AGENT_PROTOCOL_VERSION,
       '0.4'
     )
+    const DD_TRACE_PARTIAL_FLUSH_MIN_SPANS = coalesce(
+      parseInt(options.flushMinSpans),
+      parseInt(process.env.DD_TRACE_PARTIAL_FLUSH_MIN_SPANS),
```

Comment:
Is this env var name consistent across tracers?

Reply:
It was taken directly from dd-trace-java. I don't know if it's implemented in other languages.

Reply:
Fair enough.